spec/dev/server.ts

```diff
@@ -54,10 +54,12 @@ export async function start() {
         },*/
         loggerProvider
     });
+    //@ts-ignore
```

Comment:
oh, what's the error you're supressing here?

Reply:
So there are 3 type errors in this file in my env. in order:

```
Argument of type 'InMemoryAdapter | ArangoDBAdapter' is not assignable to parameter of type 'DatabaseAdapter'.
  Type 'ArangoDBAdapter' is not assignable to type 'DatabaseAdapter'.
    Types of property 'execute' are incompatible.
      Type '(queryTree: import("/Users/orefalo/GitRepositories/arangodb-related/cruddl/dist/src/query-tree/base").QueryNode) => Promise<any>' is not assignable to type '(queryTree: import("/Users/orefalo/GitRepositories/arangodb-related/cruddl/src/query-tree/base").QueryNode) => Promise<any>'.
        Types of parameters 'queryTree' and 'queryTree' are incompatible.
          Type 'import("/Users/orefalo/GitRepositories/arangodb-related/cruddl/src/query-tree/base").QueryNode' is not assignable to type 'import("/Users/orefalo/GitRepositories/arangodb-related/cruddl/dist/src/query-tree/base").QueryNode'.
            Types of property 'equals' are incompatible.
              Type '(other: import("/Users/orefalo/GitRepositories/arangodb-related/cruddl/src/query-tree/base").QueryNode) => boolean' is not assignable to type '(other: import("/Users/orefalo/GitRepositories/arangodb-related/cruddl/dist/src/query-tree/base").QueryNode) => boolean'.
                Types of parameters 'other' and 'other' are incompatible.
                  Type 'import("/Users/orefalo/GitRepositories/arangodb-related/cruddl/dist/src/query-tree/base").QueryNode' is not assignable to type 'import("/Users/orefalo/GitRepositories/arangodb-related/cruddl/src/query-tree/base").QueryNode'.
                    Property 'fieldEquals' is protected but type 'QueryNode' is not a class derived from 'QueryNode'.
```

```
Argument of type 'Model' is not assignable to parameter of type 'Model & Model'.
  Type 'import("/Users/orefalo/GitRepositories/arangodb-related/cruddl/src/model/implementation/model").Model' is not assignable to type 'import("/Users/orefalo/GitRepositories/arangodb-related/cruddl/dist/src/model/implementation/model").Model'.
    Types have separate declarations of a private property 'input'.
```

```
Argument of type 'InMemoryAdapter | ArangoDBAdapter' is not assignable to parameter of type 'DatabaseAdapter'.
  Type 'ArangoDBAdapter' is not assignable to type 'DatabaseAdapter'.
```

Reply:
oh that might be the case because you once built cruddl (see that some paths include `dist` and some don't. If you remove the `dist` directory, it should be fixed. I guess we could adjust something in the `tsconfig.json` to fix this

Reply:
this one, you are in full control - I tried before and couldn't find the source of the issue

Reply:
yeah you don't need to fix the `tsconfig.json`, deleting `dist` and making sure it works with the `@ts-ignore` should be fine.

Reply:
confirmed - it was the issue.

Now I understand your point.  interesting issue.

anyway stuff removed